app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,72 @@
+class QuestionsController < ApplicationController
+  def index
+    if  params[:order] == "highest"
+      @questions = Question.by_points
+    elsif params[:order] == "recent"
+      @questions = Question.recent_questions
+    elsif params[:order] == "trending"
+      @questions = Question.by_trending
+    else
+      @questions = Question.all
+    end
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def show
+    @question = Question.find(params[:id])
+    @question_voters = @question.voted
```

Comment:
I would probably not bother doing this and just de-reference @question.voted in the view.

Also the .voted method might be better named voters? - it returns a list of people who voted.